pkg/gitops/flux/flux.go

```diff
@@ -149,6 +151,78 @@ func (f *Flux) Uninstall(ctx context.Context, cluster *types.Cluster, clusterSpe
 	return nil
 }
 
+func (f *Flux) PauseClusterResourcesReconcile(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec, provider providers.Provider) error {
+	if f.shouldSkipFlux() {
+		logger.Info("GitOps field not specified, pause cluster resources reconcile skipped")
```

Comment:
nit: maybe do this for level 4?
It doesn't look like something we should print by default